### PR TITLE
Updated mod job description

### PR DESCRIPTION
**Pull request by seotts**
### Resolves:

Issue #2806, update community job description. 

Updates job description to https://docs.google.com/document/d/1TYus13MIF717HSUpcdzI5yio_fem26btmnGZG7WIpxI/edit#

### Changes:
`/jobs/moderator` now displays the new job description. 

### Test Coverage:

I don't think any tests are necessary.

### CODE REVIEW

File: src/views/jobs/moderator/moderator.jsx [hunk 1]
```diff
@@ -9,65 +9,86 @@ const Moderator = () => (
...
             </p>
             <h3>Responsibilities:</h3>
             <ul>
                 <li>
-                    Participate actively in the Scratch online community as
-                    a mentor and resource for youth
+                    Promote the values and core ideas of the Scratch
+                    project (such as inclusiveness, creative collaboration,
+                    and constructive feedback)
+
```
Comment (towerofnix) [inline src/views/jobs/moderator/moderator.jsx]: Just for code-style, there shouldn't be an extra line here or on line 39.

**Comment by thisandagain** [comment]
@seotts Can you please resolve the conflicts on this and address the code style nit from @towerofnix?

**Comment by seotts** [comment]
@thisandagain @towerofnix Just fixed it, thanks

**Comment by seotts** [comment]
@rschamp Ah cool, thanks for noticing that! If it's good to merge, could you do so? (I can't merge if checks are unsuccessful.)

**Pull request merged**